roles/nova/vars/main.yml

```
@@ -76,14 +76,16 @@ _nova_helm_values:
         connection_recycle_time: 10
         max_pool_size: 1
       filter_scheduler:
+        available_filters: nova_scheduler_filters.FailureDomainFilter
```

This removes all the existing filters

```suggestion
        available_filters:
          type: multistring
          values:
            - nova.scheduler.filters.all_filters
            - nova_scheduler_filters.FailureDomainFilter
```